bin/functionMetadata_original.php

```diff
@@ -153,6 +153,8 @@
 	'DateTimeImmutable::getTimestamp' => ['hasSideEffects' => false],
 	'DateTimeImmutable::getTimezone' => ['hasSideEffects' => false],
 
+	'Redis::get' => ['hasSideEffects' => true],
```

Comment:
1) Are there other Redis methods that behave the same way? I'd say yes, please provide a complete list. Alternatively you can also modify code in src/ to achieve the same result, along with some tests.
2) This change isn't sufficient, you need to run a script in bin/ to regenerate resources/functionMetadata.php

Thanks.